content/en/logs/processing/parsing.md

```diff
@@ -495,6 +496,63 @@ rule %{data::xml}
 * If the XML contains tags that have both an attribute and a sting value between the two tags, a `value` attribute is generated. For example: `<title lang="en">Harry Potter</title>` is converted to `{"title": {"lang": "en", "value": "Harry Potter" } }`
 * Repeated tags are automatically converted to arrays. For example: `<bookstore><book>Harry Potter</book><book>Everyday Italian</book></bookstore>` is converted to `{ "bookstore": { "book": [ "Harry Potter", "Everyday Italian" ] } }`
 
+### Parsing CSV
+
+Use the **csv** filter to more-easily map strings to attributes when separated by a given character (`,` by default).
+
+The csv filter is defined as `csv(headers[, separator[, quotingcharacter]])` where:
+
+* `headers`: defines the keys name separated by `,`. Keys names must start with alphabetical character and can contain any alphanumirical character in addition to `-`, `_`, `@` and `$`.
+* `separator`: defines separators used to separate the different values. Only one character is accepted. Default: `,` . Note use `tab` to represent the tabulation character.
+* `quotingcharacter`: defines the quoting character. Only one character is accepted. Default: `"` 
+
+**Note**:
+
+* Values containing a separator character must but quoted.
+* Quoted Values containing a quoting character must be escaped with a quoting characters. ex: `""` represent `"`.
+* If the log doesn't contain the same number of value as the number of keys in the header, the csv parser will match the first ones. 
+
+
+**Log**:
+
+```text
```

Comment:
We've switched to new code blocks so users have an easy copy button / better view! :)


{{< code-block lang="json" filename="block.json" >}}
your code example here
{{< /code-block >}}

Reply:
Done